app/adapters/activity.js

```diff
@@ -0,0 +1,7 @@
+import ApplicationAdapter from './application';
```

Comment:
Doesn't feedbackfruits-ui-activities already provide an Activity model that has the endpoints `/v1/activities`? Can't we use that?